core/services/vrf/v2/listener_v2_log_processor.go

```diff
@@ -565,55 +565,53 @@ func (lsn *listenerV2) enqueueForceFulfillment(
 	}
 
 	// fulfill the request through the VRF owner
-	err = lsn.q.Transaction(func(tx pg.Queryer) error {
```

Comment:
Weird this was a transaction in the first place 🤔

Reply:
Yeah, maybe it was just nudged to a confused state from edits over time :shrug: